docs/docs/cloud/quick_start.md

```diff
@@ -175,36 +213,26 @@ curl --request POST \
         "messages": [
             {
                 "role": "user",
-                "content": "How are you?"
+                "content": "What is the weather in NYC?"
```

Comment:
How is the result? does it return the  correct weather in NYC?

Reply:
yea it's a bit verbose, will just trim it